LiveSplit.HollowKnight.csproj

```diff
@@ -148,6 +146,7 @@
     <EmbeddedResource Include="PlayerData\V1032.txt" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="LocalBuildProperties.props" />
```

Comment:
I'm curious, why did you add this line? Does it solve a problem that I hadn't noticed?

Reply:
It was not intentional, VS did this when I followed the comment instructions in this file.